main.tf

```diff
@@ -397,10 +425,11 @@ data "google_compute_default_service_account" "default" {
 resource "google_cloud_scheduler_job" "agentless_orchestrate" {
   count = var.regional ? 1 : 0
 
-  name        = "${var.prefix}-periodic-trigger-${local.suffix}"
+  name        = "${var.prefix}-periodic-trigger-${local.suffix}-${local.region}"
```

Comment:
I've run into issues in Azure of hitting max name length. Not sure if it's the same case in GCP, but may worth checking for some of the longer names (e.g. `northamerica-northeast2`).

Reply:
Thanks Ao. I just verified this works fine with the longest name we have here

<img width="373" alt="image" src="https://github.com/lacework/terraform-gcp-agentless-scanning/assets/1553680/4cb3c27c-23e8-4e07-9c1b-ad28bb82486b">